### ISSUE TITLE
Cannot read properties of undefined (reading 'id')

### ISSUE THREAD

**Issue by the-Jinxist** [MeilCli/slack-upload-file#855]
## Description
While running the build action, the apk gets sent to the channel but we get an error with the message: `Error: Cannot read properties of undefined (reading 'id')`

The action looks like this

```
with:
    slack_token: ***
    channel_id: C04H7GCS7R9
    file_path: build/app/outputs/flutter-apk/app-production-release.apk
    initial_comment: :red_circle: *Production* `Release mode`
  >`PR-442`: Maa 1139 order rejection flow update
  >`PR Link`: https://github.com/mrsool/accept/pull/442
  >`commit`: https://github.com/mrsool/accept/pull/442/commits/749f8234216687fb6bf8574d1f8f235084e3fea6
  *Ready for QA Testing*: `NO`
    file_path_follow_symbolic_links: true
  env:
    keyAlias: ***
    keyPassword: ***
    storePassword: ***
    PR_NUMBER: 442
    PR_TITLE: Maa 1139 order rejection flow update
    JAVA_HOME: /opt/hostedtoolcache/Java_Zulu_jdk/11.0.21-9/x64
    JAVA_HOME_11_X64: /opt/hostedtoolcache/Java_Zulu_jdk/11.0.21-9/x64
    FLUTTER_ROOT: /opt/hostedtoolcache/flutter/stable-3.13.2-x64
    PUB_CACHE: /opt/hostedtoolcache/flutter/stable-3.13.2-x64/.pub-cache
Error: Cannot read properties of undefined (reading 'id')
```
<img width="1159" alt="Screenshot 2023-11-02 at 12 10 44" src="https://github.com/MeilCli/slack-upload-file/assets/35970737/c30f805a-7209-41f4-a3fa-a234e5093457">

## Environment
Flutter app
`runs-on: ubuntu-latest`

**Comment by the-Jinxist** [comment]
There's no `id` variable on the github actions at all.

**Comment by MeilCli** [comment]
Thanks reporting issue and checking same issue.
I just published [v3.1.0](https://github.com/MeilCli/slack-upload-file/releases/tag/v3.1.0), and it contains fixing this problem.
